src/SWOTRiver/SWOTRiverEstimator.py

```diff
@@ -998,14 +982,7 @@ def process_reach(self,
         reach_stats['slp_nr'] = np.float32(reach_stats['slp_nr'])
         reach_stats['nr_rsqrd'] = np.float32(reach_stats['nr_rsqrd'])
         reach_stats['nr_mse'] = np.float32(reach_stats['nr_mse'])
-
-        # trap out of range / missing data
```

Comment:
This was removed for some reason, its a int32 will fill value -9999 before this point so I have to trap out of bounds else it will drop MSB to fit:
```
In [3]: np.uint8(-9999)
Out[3]: 241
```